labelbox/schema/project.py

```diff
@@ -150,7 +150,7 @@ def members(self) -> PaginatedCollection:
         """
         id_param = "projectId"
         query_str = """query ProjectMemberOverviewPyApi($%s: ID!) {
-             project(where: {id : $%s}) { id members(skip: %%d first: %%d){ id user { %s } role { id name } }
+             project(where: {id : $%s}) { id members(skip: %%d first: %%d){ id user { %s } role { id name } accessFrom }
```

Comment:
You need to add the `accessFrom` attribute to ProjectMember. Otherwise it will be ignored.

Reply:
I think this is a string. So you can just add Field.String("accessFrom") here https://github.com/Labelbox/labelbox-python/blob/dc44aa8d304afe939a3afa2f41b66fc0268b4c5c/labelbox/schema/project.py#L1363

Reply:
done